rdkit/Chem/PandasTools.py

```diff
@@ -314,7 +320,7 @@ def ChangeMoleculeRendering(frame=None, renderer='PNG'):
 
 
 def LoadSDF(filename, idName='ID', molColName='ROMol', includeFingerprints=False,
-            isomericSmiles=False, smilesName=None, embedProps=False):
+            isomericSmiles=True, smilesName=None, embedProps=False, removeHs=True, strictParsing=True):
```

Comment:
I think defaulting to `isomericSmiles=True` is exactly right, but please add a `Backwards Incompatible Changes` entry to the release notes for the change (like this section: https://github.com/rdkit/rdkit/blob/master/ReleaseNotes.md#backwards-incompatible-changes)